src/test/scala/com/bwsw/cloudstack/entities/requests/UserCreateRequestTestSuite.scala

```diff
@@ -0,0 +1,77 @@
+/*
+* Licensed to the Apache Software Foundation (ASF) under one
+* or more contributor license agreements. See the NOTICE file
+* distributed with this work for additional information
+* regarding copyright ownership. The ASF licenses this file
+* to you under the Apache License, Version 2.0 (the
+* "License"); you may not use this file except in compliance
+* with the License. You may obtain a copy of the License at
+*
+* http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing,
+* software distributed under the License is distributed on an
+* "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+* KIND, either express or implied. See the License for the
+* specific language governing permissions and limitations
+* under the License.
+*/
+package com.bwsw.cloudstack.entities.requests
+
+import java.util.{TimeZone, UUID}
+
+import br.com.autonomiccs.apacheCloudStack.client.ApacheCloudStackApiCommandParameter
+import org.scalatest.FlatSpec
+
+import scala.collection.JavaConverters._
+
+class UserCreateRequestTestSuite extends FlatSpec {
+  val settings = UserCreateRequest.Settings(
+    accountName = "accountName",
+    email = "test@example.com",
+    firstName = "fn",
+    lastName = "ln",
+    password = "password",
+    username = "test"
+  )
+
+  val defaultParameters = Set[ApacheCloudStackApiCommandParameter](
+    new ApacheCloudStackApiCommandParameter("response","json"),
+    new ApacheCloudStackApiCommandParameter("account", settings.accountName),
+    new ApacheCloudStackApiCommandParameter("email", settings.email),
+    new ApacheCloudStackApiCommandParameter("firstname", settings.firstName),
+    new ApacheCloudStackApiCommandParameter("lastname", settings.lastName),
+    new ApacheCloudStackApiCommandParameter("password", settings.password),
+    new ApacheCloudStackApiCommandParameter("username", settings.username)
+  )
+
+  "Instance creation" should "create request with default parameters" in {
```

Comment:
it should create a request with predefined and specified (via constructor) parameters

Reply:
ok